### ISSUE TITLE
Fixed: Reflected Cross Site Scripting (XSS) vulnerability found in version(s) <= 1.3.3.

### ISSUE THREAD

**Issue by gbissland** [plausible/wordpress#159]
Not an issue, as this looks like it was fixed in 1.3.4, but am leaving it here anyway.
A good idea for everyone to check they have updated to the latest version.

Versions below 1.3.4 have been flagged for a Cross Site Scripting (XSS) vulnerability.

https://patchstack.com/database/vulnerability/plausible-analytics/wordpress-plausible-analytics-plugin-1-3-3-reflected-cross-site-scripting-xss-vulnerability

**Comment by Dan0sz** [comment]
Yes. We were made aware of this a few weeks ago and fixed it ASAP.
